internal/benchmarks/benchmarks_test.go

```diff
@@ -173,6 +173,7 @@ func BenchmarkReadSchema(b *testing.B) {
 		// We don't want this benchmark to test the network so instead we run the actual function in a tight
 		// loop within a single execution.
 		executions := 10000
+		// nolint:gosec
```

Comment:
instead of ignoring this, I believe the linter wants you to pass the executions number as an arg in `ExecContext`